### DISCUSSION TITLE
Support for pouchdb (local database)

### DISCUSSION THREAD

**Discussion by aradhya-tiwari** [prisma/prisma#16737] [Q&A]
Hello, I want a read-only local database to work with NEXTjs/SvelteKit hosted on git but SQLite and Prisma don't seem to work this way, So can we add support for pouchdb in Prisma?? 
Sorry if it sounds stupid but there is no solution matching my need. 
Thank you !!!

**Comment by nurul3101** [comment]
Hello @aradhyatiwari 👋 
I am not able to find any existing feature requests for pouchdb, can you please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for it?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
